intranet/templates/signage/pages/bus.html

```diff
@@ -98,53 +106,113 @@
     svg text.extra-small {
         font-size: 90%;
     }
+
+    .col-1 {
+        display: inline-block;
+        width: 33%;
+        height: 100%;
+        vertical-align: top;
+    }
```

Comment:
I would avoid using common names from external libraries like Bootstrap, just in case we switch later (though it's very unlikely).